reference/Mono.Android.xml

```diff
@@ -776349,7 +777654,7 @@
                   </attribute>
                 </attributes>
               </field>
-              <field name="ChoiceModeSingle" attrib="32854" fieldtype="Android.Widget.ChoiceMode" value="1">
```

Comment:
This looks like an ABI break. Should we be worried?

Reply:
They are constants.

Reply:
It's a `const`, this should be fine.